### PR TITLE
self._inverted_axes must be a list

### LINKED ISSUE #15: Cannot set calc constraints
When trying to set `calc` constraints , received `TypeError: argument of type 'NoneType' is not iterable` exception.

In this example, `fourc` is an instance of `E4CV` with specific EPICS motors.  The ipython session:

```
In [1]: fourc.calc["tth"]                                                                                                                   
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-1-b53b540f6eef> in <module>
----> 1 fourc.calc["tth"]

~/Apps/anaconda/envs/bluesky/lib/python3.7/site-packages/hkl/calc.py in __getitem__(self, axis)
    408         return CalcParameter(self._get_axis_by_name(axis),
    409                              units=self._unit_name, name=axis, inverted=axis in
--> 410                              self._inverted_axes, geometry=self._geometry)
    411 
    412     def __setitem__(self, axis, value):

TypeError: argument of type 'NoneType' is not iterable

In [2]: fourc.calc._inverted_axes is None                                                                                                   
Out[2]: True
```

### PR DESCRIPTION
**Pull request by prjemian**
fixes #15

### CODE REVIEW

File: hkl/calc.py [hunk 1]
```diff
@@ -101,7 +101,7 @@ def __init__(self, dtype, engine='hkl', sample='main', lattice=None,
         self._lock = RLock()
         self._axis_name_to_renamed = {}
         self._axis_name_to_original = {}
-        self._inverted_axes = inverted_axes
```
Comment (tacaswell) [inline hkl/calc.py]: Did you mean `self._inverted_axes or []`?

Comment (klauer) [reply hkl/calc.py]: Did you mean `inverted_axes or []`?

Comment (tacaswell) [reply hkl/calc.py]: :sheep:

Comment (prjemian) [reply hkl/calc.py]: yup, bad edit

**Pull request merged**